README.md

```diff
@@ -2,11 +2,11 @@
 [![Published on webcomponents.org](https://img.shields.io/badge/webcomponents.org-published-blue.svg)](https://beta.webcomponents.org/element/PolymerElements/paper-input)
 
 ## &lt;paper-input&gt;
-
-Material design: [Text fields](https://www.google.com/design/spec/components/text-fields.html)
-
 `<paper-input>` is a single-line text field with Material Design styling.
 
+See: [Documentation](https://www.webcomponents.org/element/PolymerElements/paper-input),
+  [Demo](https://www.webcomponents.org/element/PolymerElements/paper-input/demo/demo/index.html).
```

Comment:
FYI, these links will change in NPM world. They should redirect, but I would probably rather just the top level docs link instead of deep linking to the demo.

Reply:
yeah! i dont know how?

Reply:
i kinda wanted to be able to one-click link to the demo since that's p useful for a quick "what does this element do". will there be any canonical link to the demo in the future?

Reply:
I think it'll be the same, but not 100% sure. The other change I meant was that its `element/@polymer/paper-input` now not `element/PolymerElements/paper-input`.